docker-compose.yaml

```
@@ -9,40 +9,32 @@
-    image: nginx:alpine
-    container_name: electronyat-reverse-proxy
-    ports:
-        - "8060:80"
-    volumes:
-        - ./nginx.conf:/etc/nginx/conf.d/default.conf
-        - /var/run/docker.sock:/tmp/docker.sock:ro
-    networks:
-        - electronyat-network
-    depends_on:
-      - electronyat-shop
+        - electronyat-shop
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 6 but found 8

(indentation)